spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -603,6 +607,11 @@ public int incrementRetries() {
 		 */
 		private String charset = Charset.defaultCharset().name();
 
+		/**
+		 * Whether or not add MDC properties into message headers. Default is true to keep backward compatibility.
+		 */
+		private boolean serializeMdc = true;
```

Comment:
I don' think this property name properly reflects what it does.

Maybe `addMdcHeaders` ?

Reply:
well, it is not very accurate as while mdc has no headers. maybe addMdcToHeaders?

Reply:
`addMdcAsHeaders` ?

Reply:
looks fine